spec/swagger_schemas/attributes/id.rb

```diff
@@ -0,0 +1,5 @@
+ID_ATTRIBUTE = {
```

Comment:
wondering if all those schemas should be under a versioned folder?

Reply:
So I originally intended to put the schemas in a folder relative to their version, but when I came to construct the schema definition in the `swagger_helper` I found that made it more difficult to do it dynamically; I think having all versions in the same file/constant is going to be more straight forward when we come to add other version definitions, but happy to review it if we run into issues when we come to do that if that would make sense?

Reply:
sure we can keep for now and review as we go 👍